src/zeta/aca_zeta_programming.cpp

```diff
@@ -50,10 +51,18 @@ void ACA_Zeta_Programming::create_entry(string zeta_gateway_id, uint oam_port)
   // then add 1 after, doing both atomically
   // std::memory_order_relaxed option won't help much for x86 but other
   // CPU architecture can take advantage of it
+
+  new_zeta_cfg->oam_port = oam_port;
```

Comment:
please move this line back to below so that the comments in line 51-53 still apply for current_available_group_id.fetch_add...